cps/metadata_provider/amazon.py

```diff
@@ -0,0 +1,98 @@
+import concurrent.futures
+import requests
+from bs4 import BeautifulSoup as BS  # requirement
+import lxml     #requirement for better speed
+import cchardet #optional for better speed
```

Comment:
It it's optional, please put a try: except ImportError statement around it, so users can use it without installing cchardet